Patch104pZH/GameFilesEdited/Data/INI/Object/TechBuildings.ini

```
@@ -1143,8 +1143,8 @@ Object TechRadioStation
   RadarPriority       = STRUCTURE
 
   Body                = ActiveBody ModuleTag_04
-    MaxHealth         = 1000.0
-    InitialHealth     = 1000.0
+    MaxHealth         = 1500.0
+    InitialHealth     = 1500.0
```

```suggestion
    MaxHealth         = 1000.0
    InitialHealth     = 1000.0
```
keep tiny building as is